src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -617,6 +617,17 @@ public static bool IsSameLanguage(ProjectState project1, ProjectState project2)
             return project1.LanguageServices == project2.LanguageServices;
         }
 
+        public bool ContainsAnyReferenceToProject(ProjectId projectId)
```

Comment:
`ContainsDirectReferenceToProject`?  `ContainsAny` makes me think that it's trnasitive.

Reply:
ProjectState doesn't have the concept of transitive references. This internal helper goes with the `ProjectReferences` property and `RemoveProjectReference` method.

Reply:
> ProjectState doesn't have the concept of transitive references

Right.  But that's ambient information.  In the context of this domain that isn't necessarily clear, and just having the wording be nicer would be good.

Note: if this is suposed to go with ProjectReferences/RemoveProjectReference, i would expect this to simply be `HasProjectReference(ProjectId)`. the `ContainsAny` actually makes things more confusing.

Reply:
Renamed to `ContainsReferenceToProject` and added a comment